packages/network/lib/uri.ts

```diff
@@ -10,24 +10,24 @@ import url, { URL } from 'url'
 
 // yup, protocol contains a: ':' colon
 // at the end of it (-______________-)
-const DEFAULT_PROTOCOL_PORTS = {
+const DEFAULT_PROTOCOL_PORTS: Record<string, string> = {
   'https:': '443',
   'http:': '80',
 }
 
 const DEFAULT_PORTS = _.values(DEFAULT_PROTOCOL_PORTS)
 
-const portIsDefault = (port) => {
+const portIsDefault = (port: string | null) => {
```

Comment:
Could also improve typing here - `port: '80' | '443'` are the only valid ports, right?

Reply:
Not sure how easy/hard these types are to add - it might make other places in the code base complain.

Reply:
It seems that current behavior is correct because the purpose of this function is to check if the url uses the default port or **not**. We can use other ports like `3000`, `8080` for internal sites